### ISSUE TITLE
New outgoing liaison form fails at the From contact 

### ISSUE THREAD

**Issue by rjsparks** [ietf-tools/datatracker#4610]
On submission, validation of valid from contacts fail - the field is cleared, and unless you are in very special role, the field is also disabled, so there is no recourse.

**Comment by jennifer-richards** [comment]
The `disabled` flag is set on the input - this has the effect both of disabling it (desired) and omitting its value from the POST (less desired).

**Comment by jennifer-richards** [comment]
Is there any need to show this field if it's not editable? I suspect the cleanest fix is to get rid of it and fill in the correct value automatically.

**Comment by larseggert** [comment]
See it this works? https://stackoverflow.com/questions/41807096/select2-make-it-readonly-not-disabled-from-js/55001516#55001516

**Comment by jennifer-richards** [comment]
Yes, looks like it does. Thanks @larseggert

**Comment by larseggert** [comment]
Did this get fixed in the meantime?

**Comment by jennifer-richards** [comment]
Yes, it was fixed by #4708 but did not auto-close this ticket because that went to feat/tzaware rather than main.
